BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php

```diff
@@ -67,6 +67,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('authorize_status_change.yml');
         $loader->load('authorize_edition.yml');
         $loader->load('restore_entity.yml');
+        $loader->load('dashboard.yml');
```

Comment:
In each case, the configuration that the integrator has set is not analyzed, and in each case the dashboard will contain only the few one we set.

I don't know if the merge is the best solution, but I think that a solution inbetween could be found :)